src/main/java/wellnus/exception/TokenizerException.java

```diff
@@ -0,0 +1,7 @@
+package wellnus.exception;
+
+public class TokenizerException extends WellNusException {
```

Comment:
Remember to add Javadoc -- this will cause checkstyle regression issues later on